test/MOL_1D_Linear_Convection.jl

```diff
@@ -0,0 +1,45 @@
+# 1D linear convection problem
```

Comment:
These need to be added to runtests.jl to run.

Reply:
Done.

Note: there are other tests that are currently failing (at least on my computer) when the file runtests.jl is executed. E.g. DiffEqOperators.jl/test/concretization.jl:
First Dimension: Error During Test at .../DiffEqOperators.jl/test/concretization.jl:18
  Test threw exception
  Expression: L1 * M ≈ Array(L1, size(M)) * vec(M) ≈ sparse(L1, size(M)) * M ≈ BandedMatrix(L1, size(M)) * M ≈ BandedBlockBandedMatrix(L1, size(M)) * M
...